MedicationScheduler/AuthApp/views.py

```diff
@@ -38,7 +38,7 @@ def login_view(request):
             if user is not None:
                 print(user)
                 login(request, user)
-                return redirect('/')
+                return redirect('/home')
```

Comment:
We should use the names of the URLs this will make it easier to change the actual URL later. In this case, it would be `'MedicationScheduler:home'`. Could you also change the similar instances of this in this PR?

Reply:
Actually, you will have to add an `app_name` variable to `urls.py` files in both AuthApp and MedSchedulerApp. An example of this is in my "Add Medication and Prescription" PR.

Reply:
@bew5294 done